x-pack/platform/plugins/shared/maps/public/routes/map_page/map_app/map_app_styles.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CSSObject } from '@emotion/serialize';
+
+export const mapsPluginStyles: CSSObject = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100%',
+  overflow: 'hidden',
+};
+
+export const mapFullScreenStyles = {
+  height: '100vh !important',
+};
```

Comment:
intentionally not adding the type `CSSObject` because this type is not compatible with the `style` prop 🤷, and I'm using it there because it's a dynamic style.